support/Charm/v7.0.1.patch

```diff
@@ -0,0 +1,101 @@
+diff --git a/cmake/detect-features-cxx.cmake b/cmake/detect-features-cxx.cmake
```

Comment:
Are the patches just copies from 7.0.0?

Reply:
Yes, they are just exact copies, but because of the way we identify things, there's no real way around that. Maybe symlinks? But I don't know how well symlinks work in git repos...

Reply:
Nah the copy is fine